lib/compilers/glimmer/index.js

```diff
@@ -13,14 +13,39 @@ export default class GlimmerTemplatePrecompiler extends Filter {
 
   processString(content, relativePath) {
     let specifier = getTemplateSpecifier(this.rootName, relativePath);
-    return `export default ${this.precompile(content, { meta: { specifier, '<template-meta>': true } })};`;
+    let { contentWithReplacedPragmaTag, managerId } = extractPragmaTag(content);
+    let meta = { specifier, '<template-meta>': true };
```

Comment:
`<template-meta>` is an internal (VM) brand and likely does not need to be here.